src/app/pickup/page.tsx

```diff
@@ -65,12 +109,39 @@ export default function Pickup() {
               Pick Up
             </Normal700Text>
             <Normal700Text>Name</Normal700Text>
-            <PickupContent>Ethan Auyeung</PickupContent>
+            <PickupContent>
+              {Profile?.first_name} {Profile?.last_name}
+            </PickupContent>
             <Normal700Text>Phone Number</Normal700Text>
-            <PickupContent>+1 123-456-7890</PickupContent>
-            <PickupButton day="hi" date="date" start="start" end="end" />
+            <PickupContent>{Profile?.phone_numbers}</PickupContent>
+            <div style={{ display: 'flex', justifyContent: 'space-around' }}>
+              <Normal700Text>Pick Up Time</Normal700Text>
+              <Normal700Text>Pick Up times: 10:00 AM - 12:00 PM </Normal700Text>
+            </div>
+
+            <div style={{ display: 'flex', justifyContent: 'space-around' }}>
+              {Time.map((time, index) => (
+                <PickupTimeButton
+                  key={time.id} // Replace 'index' with a unique identifier from the 'time' object
+                  isSelected={selectedPickupIndex === index}
+                  onClick={() => handleButtonClick(index)}
+                >
+                  <div>
```

Comment:
if this is the date, why is it surrounded by a div and why is there no styling? try to use the text components